go/exorelay/exorelay_test.go

```diff
@@ -126,6 +164,56 @@ func FeatureContext(s *godog.Suite) {
 		}
 		return nil
 	})
+
+	s.Step(`^I register a handler for the "([^"]*)" message$`, func(messageName string) error {
+		mapping := map[string]func(map[string]interface{}){}
+		mapping[messageName] = func(payload map[string]interface{}) {
+			handlerCalled = true
+			savedPayload = payload
+		}
+		exoInstance.BindMessageHandlers(mapping)
```

Comment:
I'd go with error checking. I think the resulting API is a lot more developer friendly. Similar to how we use query builders to build up massive SQL string constructs piece by piece.